.github/workflows/propogate-auto-merge.yml

```
@@ -0,0 +1,57 @@
+jobs:
+  propagate-label:
+    runs-on: ubuntu-latest
+
+    permissions:
+      pull-requests: write
+      issues: write
+
+    steps:
+      - name: Detect backport and original PR
+        id: extract
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)